coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
you need to add the coverage folder to the `.gitignore`

Reply:
added!